src/components/EventListCard/EventListCard.module.css

```diff
@@ -118,3 +120,81 @@ form > input {
 .checkboxdiv > div {
   width: 50%;
 }
+
+.preview {
+  display: flex;
+  flex-direction: row;
+  font-weight: 900;
+  font-size: 16px;
+  color: rgb(80, 80, 80);
+}
+.view {
+  margin-left: 2%;
+  font-weight: 600;
+  font-size: 16px;
+  color: #707070;
+}
+.flexdir {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border: none;
+}
+
+.form_wrapper {
```

Comment:
Why not use flex to centre the div here instead of absolute. I find absolute positioning to be unpredictable at times. Additionally, aligning a div to the centre seems a bit more scalable when using flex or grids.